nl_server/registry.py

```diff
@@ -33,57 +34,59 @@
 
 
 #
-# A map from specific index to embeddings stores and models.
+# A class to hold embeddings stores and models.
```

Comment:
Is there a more precise definition, and distinguish it from Env and Catalog?

Reply:
Updated